csvtotable/templates/template.j2

```diff
@@ -13,13 +13,15 @@
 
 	<script src="js/jquery.min.js" type="text/javascript"></script>
 	<script src="js/jquery.dataTables.min.js" type="text/javascript"></script>
+	<script src="js/echarts.min.js" type="text/javascript"></script>
```

Comment:
Since the included JS is around 600kb its better if this is conditional based on export options enabled.

Reply:
Make sense, will make it optional depending on the input options